### DISCUSSION TITLE
Vstream

### DISCUSSION THREAD

**Discussion by fredseventyfive** [Kodi-vStream/venom-xbmc-addons#4085] [Support & aide] [answered]
Sur Xbox série X, Clean install Kodi 20.2 clean install Vstream 2.6.1, compte Uptobox prenium plus Token install pas de fichiers, test effectué sur deux Xbox séries.
Un avis ?

**Comment by sizanic** [comment, marked as answer]
plusieurs tickets pour le même probleme 🤔

**Reply by fredseventyfive** [reply]
Le DNS réseau (sur machine ou routeur ?) c’est fait Cloudflare, même souci, j’essayerais d’envoyer un Log, après c’est assez récent 72 h environ

**Reply by sizanic** [reply]
On ne va pas tenir deux discussions sur le même sujet 🤔 

https://github.com/Kodi-vStream/venom-xbmc-addons/issues/4084  !!

**Comment by fredseventyfive** [comment]
Le DNS réseau  (sur machine  ou routeur ?) c’est fait  Cloudflare, même souci, j’essayerais d’envoyer un Log pour voir si ?
